### PR TITLE
RAC-261: Add test to check exports

### PR DESCRIPTION
**Pull request by ValentinMumble**
**Description (for Contributor and Core Developer)**

Add a test that imports everything from `src/index.ts` and compares it with every component found in `src/components` and `src/icons`

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | OK
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by StevenVAIDIE — COMMENTED** [review]
Should is possible to create a story with all icons ?

With this way :
- We have a doc page with all icons
- The current script of visual test decovery will work directly

**Comment by ValentinMumble** [comment]
> Should is possible to create a story with all icons ?
> 
> With this way :
> 
>     * We can have a doc page with all icons
> 
>     * The current script of visual test decovery will work directly

@StevenVAIDIE yes, there is a ticket for it https://akeneo.atlassian.net/browse/RAC-231

**Review by juliensnz — APPROVED** [review]

**Review by StevenVAIDIE — APPROVED** [review]

File: akeneo-design-system/jest.visual.config.js [hunk 1]
```diff
@@ -3,5 +3,6 @@ const baseConfig = require('./jest.unit.config');
 module.exports = {
   ...baseConfig,
   testMatch: ['**/?(*.)+(visual).ts?(x)'],
-  preset: 'jest-puppeteer'
+  preset: 'jest-puppeteer',
+  collectCoverage: false,
```
Comment (tseho) [inline akeneo-design-system/jest.visual.config.js:7]: :call_me_hand:

File: akeneo-design-system/src/index.unit.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+describe('Every module is exported correctly', () => {
+  const exportNames = Object.keys(Exports);
+  const components = [...getSubfolders(['src/components']), ...getIcons()];
+
+  components.forEach(component => {
+    it(`Test ${component} is exported correctly.
+    If this test is failing, export ${component} component in src/index.ts`, () =>
+      expect(exportNames).toContain(component));
+  });
+});
```
Comment (tseho) [inline akeneo-design-system/src/index.unit.ts:31]: :heart_eyes:

**Review by tseho — APPROVED** [review]

**Pull request merged**